src/electron.ts

```diff
@@ -29,10 +32,35 @@ function createWindow(): Electron.BrowserWindow {
   return win;
 }
 
-app.on('ready', () => {
-  menu();
-  createWindow();
+var httpOptions = {
+  host: 'dl.bleenco.io',
+  port: 12000,
+  path: '/update'
+};
 
+http.get(httpOptions, function(resp){
```

Comment:
can you switch this function to be inside `helpers.ts` file and export it there, I think this function shouldn't be directly in main `electron.ts` file.